samples/TEE_aware/gencreds/gencreds.c

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/syscall.h>
+#include <unistd.h>
+#include <stdint.h>
+#include <myst/tee.h>
```

Comment:
Should have a comment saying "tee.h" is from Mystikos installation.

Reply:
added